package.json

```diff
@@ -50,7 +50,8 @@
     "test:e2e": "node tests/e2e/testRunner.js --development",
     "gh-actions-unused-styles": "./.github/scripts/findUnusedKeys.sh",
     "workflow-test": "./workflow_tests/scripts/runWorkflowTests.sh",
-    "workflow-test:generate": "node workflow_tests/utils/preGenerateTest.js"
+    "workflow-test:generate": "node workflow_tests/utils/preGenerateTest.js",
+    "setup-https": "mkcert -install && mkcert -cert-file config/webpack/certificate.pem -key-file config/webpack/key.pem new.expensify.com.dev localhost 127.0.0.1"
```

Comment:
@hayata-suenaga can you confirm the hostnames here match what you're looking for, or do we need to add more?

Reply:
yes `new.expensify.com.dev` is correct!